### PR TITLE
BCI: fix java memory test

### PR DESCRIPTION
**Pull request by ilausuch**
Ticket: https://progress.opensuse.org/issues/126863
VR: 
- http://openqa.suse.de/tests/10825758 (openjsk_devel the test is here now)
- http://openqa.suse.de/tests/10825759 (openjdk the test is removed from here)

### CODE REVIEW

File: tests/test_openjdk.py [hunk 1]
```diff
@@ -174,7 +174,7 @@ class TestExtendedParams:
                 expected_strings=["Iteration: (2)"],
                 expected_err_strings=["OutOfMemoryError"],
                 expected_exit_status=[1],
-                java_params="-Xmx10M",
+                java_params="-Xmx10M -Xlint:unchecked",
```
Comment (dcermak) [inline tests/test_openjdk.py:177]: Could you please explain in the commit message what & why you have done this change?

**Review by dcermak — CHANGES_REQUESTED** [review]

**Review by dcermak — CHANGES_REQUESTED** [review]
This does not solve the problem:
```
Unrecognized option: -Xlint:unchecked
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.
```

**Comment by ilausuch** [comment]
Ready

File: tests/test_openjdk_devel.py [hunk 2]
```diff
@@ -130,3 +130,26 @@ def test_compile(container, java_file: str):
...
+        [0], cmd
+    )
+
+    cmd = f"java -Xmx10M MemoryTest"
+    testout = container.connection.run_expect(
+        [1], cmd
+    )
+
+    assert "OutOfMemoryError" in testout.stderr
+    assert "1" in testout.stderr
```
[the remark this answers is not in the document]
Comment (m-dati) [reply tests/test_openjdk_devel.py:155]: Also a simple suggestion, to run tidy checks in your local, before to push :-)

[the remark this answers is not in the document]
Comment (ilausuch) [reply tests/test_openjdk_devel.py:152]: Thanks

File: tests/test_openjdk_devel.py [hunk 3]
```diff
@@ -130,3 +130,22 @@ def test_compile(container, java_file: str):
...
+
+@pytest.mark.parametrize(
+    "container",
+    CONTAINER_IMAGES_EXTENDED,
+    indirect=["container"],
+)
+def test_jdk_memory_error(container):
+    """Ensure that the environment variable ``JAVA_VERSION`` equals the output
+    of :command:`javac --version` and :command:`java --version`
+
```
Comment (dcermak) [inline tests/test_openjdk_devel.py:143]: This description does not match what is actually being tested.

**Review by dcermak — CHANGES_REQUESTED** [review]
The documentation needs to be corrected, otherwise LGTM

**Review by dcermak — APPROVED** [review]
LGTM, thank you!

**Review by m-dati — APPROVED** [review]

**Pull request merged**